docs/DeveloperGuide.md

```diff
@@ -155,12 +155,45 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add Feature
+
+#### Implementation
+
+The `add` command allows users to add students' details into the list.
+
+Given below is an example usage scenario of `add` command:
+
+Step 1. Assume the user has some existing students in the `UniquePersonList`.
+![AddState1](images/AddState1.png)
+
+Step 2. The user executes `add id/A0123456X n/John e/e0123456@u.nus.edu g/A` command to add the student into the list.
```

Comment:
Just a minor note, but I think it would be good to keep the grammar consistent and correct. For example, if you write that the `add` command _calls_ `LogicManager#execute()` then perhaps you might want to use _invokes_, _returns_, etc. in subsequent lines. Please try and use the correct tense too (line 174, for example). 

I don't see much issues apart from that.

Reply:
Ok. I have made the changes.